extensions/postgres/schema/xdb_sys_schema.sql

```diff
@@ -37,15 +37,23 @@ CREATE TABLE xdb_sys_async_state_executions(
    PRIMARY KEY (process_execution_id, state_id, state_id_sequence)
 );
 
-CREATE TABLE xdb_sys_worker_tasks(
+CREATE TABLE xdb_sys_immediate_tasks(
     shard_id INTEGER NOT NULL, -- for sharding xdb-local-mq
     task_sequence bigserial,   
     --
-    task_type SMALLINT NOT NULL, -- 1: waitUntil 2: execute
-    process_execution_id uuid, -- for looking up xdb_sys_async_state_executions
+    task_type SMALLINT NOT NULL, -- 1: waitUntil 2: execute 3: localQueueMessage
+    -- the uuid represents various UUIDs depending on the `task_type`:
+    -- if the `task_type` is waitUntil or execute, the value corresponds to the `process_execution_id`.
+    -- if the `task_type` is localQueueMessage, the value corresponds to the message `dedup_id`.
+    uuid uuid,
```

Comment:
localQueueMessage actually also need a process exection id as well. 

Maybe put the dedup_id into info field, also add a new field

Reply:
Related to the comment xdb-apis PR, a single task could be for multiple messages -- so it's more reasonable to me that the dedupId should live in the info field. 

I think keep prc_exe_id is quite important -- for a very long time, I can for see that a task is always associated with a process execution. Some tasks may not have state_id/state_id_seq, like the localMsgReceived that you are adding. but process_execution_id will be a critical part for any tasks. 

In the very long future, when we started to design "global queue", we may need to think about tasks that associated with more than one process executions(as global queue is shared acorss proc exes). At that time, I guess this proc exe could be empty.

Reply:
Good idea. I think we can put the `queue_name`, `dedup_id` and `payload` of a message into info, so that we don't need to add another field in the table.